wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
 
-### [Storyboarding](Storyboarding "wikilink")
+### Hit Sounds
 
-If anything weird in the storyboard happens (bad rendering of pictures, emptiness, etc.) it is a good idea to tell the storyboarder (who may or may not be the mapper) so that he or she can fix it. The Storyboard generally should not exceed an S.B. Load of 5.0x. You can check out the Storyboard by pressing F2 or clicking the "Design" tab at the top of the screen in [editor](editor "wikilink"). The dimensions of the elements should not exceed the default size.
+_See also: [Hit Sounds](/wiki/Hit_Sounds)._
 
-### Preview Point
+Hit sounds are the noise the game makes when the player successfully taps a [hit object](/wiki/hit_object).
+All maps should have a sufficient amount of hit sounds whether using custom or default hit sounds; otherwise, they cannot be ranked.
+
+Sometimes, hit sounds can sound awkward or weird or they are too quiet or too loud.
+Cases like these may confuse the player.
+
+Using a volume that cannot be heard, in most cases, is unrankable.
+Something like this is noteworthy to pay attention and point out.
+
+### Kiai Time
+
+_See also: [Kiai Time](/wiki/Kiai_Time)._
+
+Kiai time should be used when make sense and should not be abused.
+It is generally said to only use it where the music reaches some "epic" climax or during a chorus.
+
+In addition to this, kiai time should generally be the same in all difficulties.
+The expection to this is that [osu!taiko](/wiki/osu!taiko) [difficulties](/wiki/difficulties) may differ from [osu!standard](/wiki/osu!standard) difficulties.
+
+If the kiai time differently in any way per each difficulty, it is not a bad idea to ask whether if this was intentional or accidental.
 
-All maps must have a preview point. A preview point will be denoted by a yellow-ish line on the timeline at the bottom of the [editor](editor "wikilink"). You can also check if a map has a preview point by going into AIMod (Ctrl+Shift+A) and checking under the "Style" tab. This preview point must also be the same in all difficulties.
+### Song Setup
 
-### File Organization
+_See also: [Song Setup](/wiki/Song_Setup)._
 
-The bit rate of the song should lie between 128 to 192 kbps. Also, one empty .osb may exist but they must be deleted when more than one is present. Check if there are any file unrelated to the [beatmap](beatmap "wikilink") itself (excluding the thumb.db as this is the metafile automatically generated by the computer system).
+The actual song setup settings does not impact game play, but it is important to give correct names.
+The artist name, title of the song, source (if applicable), and tags should all be the same in each difficulty.
 
-What to avoid in your mod post
-------------------------------
+Audio lead-in times are usually the same, but can differ depending on when the first note is played.
+The `Enable Countdown` checkbox should be either enabled or disabled for all difficulties.
```

Comment:
other points:
- combo colors should be the same on the same background of each a mapset (guest mappers are an exception, but generally: difficulties by the same person and with the same background should use the same colors)
- Maps with widescreen storyboards should have the setting in `Design` -> `Misc. Toggles` -> `Widescreen Support` Toggled to avoid being cut off in 16:9 resolutions
- maps with storyboards that can cause epilepsy (frequent strobes over 3Hz) have to enable `Epilepsy Warning`
- if a map has breaks then the setting `Letterbox during breaks` should be consistent thoughout a mapset
- The `Preferred Skin` should be set to `User's preference (no change)` and occasionally is set by mappers to use their own skin for editor testing and then forgotten